<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1099</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="7">
     <widget class="QComboBox" name="selectAlg">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Douglas-Peucker</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LTTB</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>     Reading time:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="13">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>     Plotting time:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>73</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="8">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>47</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="14" colspan="2">
     <widget class="QLabel" name="nData1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="13">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Amount of data:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="13">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelPath">
      <property name="text">
       <string> Path:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="14" colspan="2">
     <widget class="QLabel" name="plottingTime2">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="lableDate">
      <property name="text">
       <string> Date:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="13">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>     Plotting time:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="13" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>218</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="15">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>73</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="13">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="14" colspan="2">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="15">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>72</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="10">
     <widget class="QPushButton" name="btnPlus">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="13">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Amount of data:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="btnSetDateTime">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Set date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="14">
     <widget class="QLabel" name="plottingTime1">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="4" colspan="12">
     <widget class="QCustomPlot" name="plot2" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5" colspan="12">
     <widget class="QCustomPlot" name="plot1" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>180</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="5">
     <widget class="QDateTimeEdit" name="dteEnd">
      <property name="displayFormat">
       <string>dd.MM.yyyy HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="5" column="13">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>73</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="14" colspan="2">
     <widget class="QLabel" name="nData2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="15">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>73</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="13" colspan="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>218</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="1" colspan="5">
     <widget class="QDateTimeEdit" name="dteStart">
      <property name="displayFormat">
       <string>dd.MM.yyyy HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QLabel" name="timeReading">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="12" column="13" colspan="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Ignored</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>218</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="6">
     <widget class="QLabel" name="LabelSelectAnAlgorithm">
      <property name="text">
       <string>     Select an algorithm:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="9">
     <widget class="QPushButton" name="btnMinus">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="3">
     <widget class="QLineEdit" name="lePath">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="btnSetPath">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Set Path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
